### ISSUE TITLE
FluentBit exporter

### ISSUE THREAD

**Issue by SergeyKanzhelev** [open-telemetry/opentelemetry-go-contrib#386]
Define a data format and create an exporter for traces and metrics from OpenTelemetry to FluentBit. This will allow customers to use existing log delivery pipeline to upload telemetry collected by the rich set of various OpenTelemetry SDKs. Pipeline we want to enalbe is SDK->FluentBit->OpenTelemetry Collector->Jaeger to see traces.

This issue is features as a [Community Bridge](https://github.com/cncf/mentoring/blob/master/communitybridge/2020/q3-q4/project_ideas.md#opentelemetry-to-fluentbit-exporter) project for Q3-Q4 2020.

(replaces https://github.com/open-telemetry/opentelemetry-dotnet-contrib/issues/31)

**Comment by MrAlias** [comment]
@SergeyKanzhelev is there a particular person that the Community Bridge project had in mind to work on this or is this open for anyone to work on this?

**Comment by Syn3rman** [comment]
Hey @MrAlias, I will be working on this :)

**Comment by SergeyKanzhelev** [comment]
@MrAlias sorry for the late reply, I was sick last week. Yes, @Syn3rman will be working on it. Please let us know if you have some preferences on the process and potential timelines this work can interfere with. I hope it's OK to work on this exporter in this repository and do it with the incremental updates. Maybe it can be excluded from the release while still in this repository. If not, we will find a proper place and will bring it back here once it's complete.

**Comment by MrAlias** [comment]
> Please let us know if you have some preferences on the process and potential timelines this work can interfere with. I hope it's OK to work on this exporter in this repository and do it with the incremental updates. Maybe it can be excluded from the release while still in this repository.

Yeah, that all sounds good :+1: as long as we can follow the [existing pattern](https://github.com/open-telemetry/opentelemetry-go-contrib/blob/master/exporters/metric/cortex/go.mod) where the exporters are their own Go modules we should be able to isolate this development well.
